BonApp.js

```diff
@@ -20,7 +18,7 @@ render: function(output) {
             '</div>',
           '</div>',
           '<div id="spinner">',
-            '<img id="spinnerImg" src="BonApp-widget/img/spinner.gif"></img>',
+            '<img id="spinnerImg" src="BonApp-Widget/img/spinner.gif"></img>',
```

Comment:
Lowercase?

Reply:
Currently downloads as uppercase. This is a fix, not a change.